### PR TITLE
Add localisation strings for FIDO error messages.

### PR DESCRIPTION
**Pull request by elibon99**

### CODE REVIEW

File: lib/fido/views/reset_dialog.dart [hunk 1]
```diff
@@ -97,7 +97,15 @@ class _ResetDialogState extends ConsumerState<ResetDialog> {
                           final String errorMessage;
                           // TODO: Make this cleaner than importing desktop specific RpcError.
                           if (e is RpcError) {
-                            errorMessage = e.message;
+                            if (e.status == 'connection-error') {
+                              errorMessage = l10n.l_failed_connecting_to_fido;
+                            } else if (e.status == 'invalid-command') {
```
Comment (AdamVe) [inline lib/fido/views/reset_dialog.dart:102]: It smells to me that for 'invalid-command' we show that the inserted key is wrong. Is the 'invalid-command' sent from the helper only in this situation and if not, could we have more specific status, for example 'key-mismatch-error', to be used here instead?

File: helper/helper/fido.py [hunk 2]
```diff
@@ -164,7 +172,11 @@ def reset(self, params, event, signal):
         self.ctap = Ctap2(connection)
         if target != _ctap_id(self.ctap):
             raise ValueError("Re-inserted YubiKey does not match initial device")
```
[the remark this answers is not in the document]
Comment (elibon99) [reply helper/helper/fido.py]: I agree! I pushed a commit that implements a more specific exception class (`KeyMismatchException`) for this type of error.

File: lib/fido/views/reset_dialog.dart (hunk 1 above)
Comment (elibon99) [reply lib/fido/views/reset_dialog.dart]: I pushed a commit that addresses this.

**Review by AdamVe — APPROVED** [review]
LGTM

**Pull request merged**